### DISCUSSION TITLE
create pipe的内部实现

### DISCUSSION THREAD

**Discussion by scpankai** [apache/iotdb#10488] [Q&A]
iotdb可以通过create pipe相关命令实现iotdb数据库之间数据的增量同步。
  我想研究create pipe内部是如何将数据发送给“目标端”的，他们的通讯协议是什么样的？相关的代码应该看哪里呢？

**Comment by wanghui42** [comment]
https://github.com/apache/iotdb/tree/master/iotdb-core/datanode/src/test/java/org/apache/iotdb/db/pipe
